cmd/exo/run.go

```diff
@@ -33,6 +33,7 @@ If a workspace does not exist, one will be created in the current directory.
 		ctx := newContext()
 		ensureDaemon()
 		cl := newClient()
+		logger := logging.CurrentLogger(ctx)
```

Comment:
Weren't we trying to avoid context-global state?

Reply:
Yes, but I needed some way to pass it to HTTP requests. It's explicit pretty much everywhere else & I didn't change `newContext` to something like `newDefaultDependencies()` or whatever to keep the change smaller.